test/protocol/OfferHandlerTest.js

```diff
@@ -788,7 +787,7 @@ describe("IBosonOfferHandler", function () {
           ).to.revertedWith(RevertReasons.INVALID_DISPUTE_RESOLVER);
         });
 
-        it("For absolute zero offer, specified dispute resolver is not active", async function () {
+        it.skip("For absolute zero offer, specified dispute resolver is not active", async function () {
```

Comment:
maybe add a `// TODO - revisit when account deactivations are supported` above the test.

Reply:
done